ci/azure-pipelines-docs-integration.yml

```diff
@@ -107,44 +107,3 @@ stages:
             AWS_ACCESS_KEY_ID: $(AWS_ACCESS_KEY_ID)
             AWS_SECRET_ACCESS_KEY: $(AWS_SECRET_ACCESS_KEY)
             AWS_DEFAULT_REGION: $(AWS_DEFAULT_REGION)
-
-    - job: test_docs_multiple_database
-      timeoutInMinutes: 30
-      condition: or(
-          eq(stageDependencies.scope_check.changes.outputs['CheckDocsDependenciesChanges.DocsDependenciesChanged'], true),
-          eq(stageDependencies.scope_check.changes.outputs['CheckDocsChanges.DocsChanged'], true),
-          eq(variables.isDevelop, true),
-          eq(variables.isManual, true)
-        )
-      variables:
-        python.version: '3.8'
-      services:
-        mysql: mysql
-        postgres: postgres
-
-      steps:
-        - task: UsePythonVersion@0
-          inputs:
-            versionSpec: '$(python.version)'
-          displayName: 'Use Python $(python.version)'
-
-        - script: |
-            pip install --constraint constraints-dev.txt ".[test, mysql, postgresql]" pytest-azurepipelines
-          displayName: 'Install dependencies'
-
-        - script: |
-            printf 'Waiting for MySQL database to accept connections'
-            until mysql --host=localhost --protocol=TCP --port=3306 --user=root --password='' --execute "SHOW DATABASES"; do
-              printf '.'
-              sleep 1;
-            done;
-          displayName: Wait for database to initialise
-
-        - script: |
-            echo "SET GLOBAL sql_mode=(SELECT REPLACE(@@sql_mode,'ONLY_FULL_GROUP_BY',''));" > mysql_setup_script.sql
-            mysql --host=localhost --protocol=TCP --port=3306 --user=root --password='' --reconnect < mysql_setup_script.sql
-          displayName: 'Configure mysql'
-
-        - script: |
-            pytest -v --docs-tests --mysql --postgresql tests/integration/test_script_runner.py
-          displayName: 'pytest'
```

Comment:
This test is only for [`cross_table_comparison`](https://github.com/great-expectations/great_expectations/blob/f4b300ff78e8d077cd6a4105d5cc037ca68e0d23/tests/integration/test_definitions/multiple_backend/integration_tests.py#L33), which is already being run in CI/CD
![Screenshot 2023-08-10 at 2 27 28 PM](https://github.com/great-expectations/great_expectations/assets/25670697/157cd0c4-27ba-4be3-8d94-e41e27fbd6b8)